ci/authn-k8s/test_gke_entrypoint.sh

```diff
@@ -54,6 +54,10 @@ function finish {
   echo '-----'
   kubectl --ignore-not-found=true delete namespace $CONJUR_AUTHN_K8S_TEST_NAMESPACE
 
+  echo 'Removing Conjur log PersistentVolume'
+  echo '-----'
+  kubectl delete --ignore-not-found=true pv $LOG_VOLUME_NAME &
```

Comment:
I'm not sure if the PVC will be removed automatically with the pods, please can you check and add a remove line for the PVC if required?

Reply:
the PVCs are removed when we remove the namespace

Reply:
Ah yes thanks.